portfolio/src/main/webapp/piano.js

```diff
@@ -0,0 +1,22 @@
+/*
+* Plays audio by key number
+*/
+
+function playNote(e) {
+  var audio = document.querySelector(`audio[data-key = "${e}"]`);
+  audio.currentTime = 0;  
```

Comment:
Why is this set to 0? You can write a comment explaining the reason.

Reply:
Because if someone wants to press the key second time before previous audio has ended, it should play audio again.

Reply:
I meant to add this as a comment in the code so it's clear in the future. If you think this is clear enough then I'll approve the change.

Reply:
oh, I understood you incorrectly, i'll fix it now